editor/src/components/sidebar/imp.rs

```diff
@@ -16,10 +16,12 @@ impl ObjectSubclass for EchidnaSidebar {
     type Type = super::EchidnaSidebar;
     type ParentType = gtk::Box;
 
+    #[inline]
```

Comment:
I'm not sure whether we can inline a function in an implementation block. I will try to see whether Rustc will love it or not, do some QA things.

Perhaps I should also ask the folks at GTK Rs about this, but I don't knoww.

Reply:
@null8626 Does Rustc allow this? I think for basic type-checking, you don't need GTK installed in your system.

Reply:
Yes

Reply:
i tested it first on a test file before creating this type of patches.